tdrs-backend/Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN apt-get -y update
 RUN apt-get -y upgrade
 # Install a new package:
 RUN apt-get install -y gcc && apt-get install -y graphviz && apt-get install -y graphviz-dev
+RUN apt-get install postgresql-client -y
```

Comment:
Is this a convenience add?

Reply:
yes, it is. Kinda unrelated to this PR but is nice to have